src/crate/client/sqlalchemy/doctests/reflection.txt

```diff
@@ -17,6 +17,14 @@ List all tables::
     >>> set(['checks', 'cluster', 'jobs', 'jobs_log']).issubset(inspector.get_table_names(schema='sys'))
     True
 
+List all views::
+
+    >>> inspector.get_view_names()
+    []
+
+    >>> inspector.get_view_names(schema='sys')
```

Comment:
schema `sys` will never contain any view as its an internal schema.
```suggestion
    >>> inspector.get_view_names(schema='my_schema')
```

Reply:
I've added 6f885e7cf34 to improve the lack of tests on this topic, which also removes inquiring VIEWs from the internal `sys` schema.